internal/storage/bucket_handle_test.go

```diff
@@ -168,6 +168,44 @@ func (t *BucketHandleTest) TestNewReaderMethodWithInvalidGeneration() {
 	AssertEq(nil, rc)
 }
 
+func (t *BucketHandleTest) TestNewReaderMethodWithCompressionEnabled() {
+	rc, err := t.bucketHandle.NewReader(context.Background(),
+		&gcs.ReadObjectRequest{
+			Name: TestObjectName,
```

Comment:
in these two tests we are trying to read the same file. is it possible to read compressed object via fake gcs server or atleast with contentEncoding propoerty set to gzip

Reply:
Done. Added a new gzip object in fakestorage with content-encoding gzip and updated all the tests accordingly.